boot/freeldr/freeldr/ntldr/winldr.c

```diff
@@ -1267,6 +1261,15 @@ LoadAndBootWindowsCommon(
 
     TRACE("Hello from paged mode, KiSystemStartup %p, LoaderBlockVA %p!\n",
           KiSystemStartup, LoaderBlockVA);
+        
+    /* Copy the page table from paged mode. */
+    if (AcpiTableSource)
+    {
+        /* This is required for 64bit OSes which can throw the BIOS Provided ACPI Tables too high. */
```

Comment:
Are there conditions we could add to skip cloning when not needed?

Reply:
let's not